### PR TITLE
fix: update handling for pages that should not show promotions or the…

### PR DESCRIPTION
**Pull request by mcstover**
…ir disclaimers

Ensures promos and disclaimers are hidden on new instant donation and lending pages.

### CODE REVIEW

**Comment by mcstover** [comment]
For hotfix today.

File: src/components/WwwFrame/DisclaimersContentful.vue [hunk 1]
```diff
@@ -86,11 +91,13 @@ export default {
 
 				// gather all inactive promo banners by their start and end dates
 				const inactivePromoBanners = uiGlobalPromoSetting?.fields?.content?.filter(promoContent => {
+					// exclude items that are not global-promo-banners
+					const isGlobalPromo = promoContent?.sys?.contentType?.sys?.id === 'globalPromoBanner';
+					if (!isGlobalPromo) return false;
```
Comment (mcstover) [inline src/components/WwwFrame/DisclaimersContentful.vue:96]: We had someone trying to add an image in contentful which was blowing things up.

File: src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue [hunk 2]
```diff
@@ -50,7 +51,15 @@ export default {
...
-				'/donate/support-kiva'
+				'/confirm-instant-donation/*',
+				'/donate/support-kiva',
+				'/error',
+				'/instant-donation-thanks',
+				'/join-team',
+				'/register/social',
+				'/possibility/giving-tuesday',
+				'/possibility/12-days-of-lending',
+				'/possibility/year-end'
```
Comment (mcstover) [inline src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue:62]: Thinking about moving this list to the new urlUtils file. Thoughts?

**Review by BoulderBrains — APPROVED** [review]
👀 👍

File: src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue (hunk 2 above)
Comment (BoulderBrains) [reply src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue:62]: +1 for putting this exclude list into urlUtils, seems like that's a good place for them to live.

File: src/pages/Checkout/CheckoutPage.vue [hunk 3]
```diff
@@ -319,13 +319,13 @@ export default {
...
-							kexpv: 'a'
-						}
-					});
+					// return Promise.reject({
+					// 	path: '/basket',
+					// 	query: {
+					// 		kexpn: 'checkout_beta.minimal_checkout',
+					// 		kexpv: 'a'
+					// 	}
+					// });
```
Comment (emuvente) [inline src/pages/Checkout/CheckoutPage.vue:328]: Was this meant to be commented out?

File: src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue (hunk 2 above)
Comment (emuvente) [reply src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue:62]: +1

File: src/pages/Checkout/CheckoutPage.vue (hunk 3 above)
Comment (mcstover) [reply src/pages/Checkout/CheckoutPage.vue:328]: oof not yet! Upcoming work.

File: src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue (hunk 2 above)
Comment (mcstover) [reply src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue]: moved and amended

File: src/pages/Checkout/CheckoutPage.vue (hunk 3 above)
Comment (mcstover) [reply src/pages/Checkout/CheckoutPage.vue]: reverted

**Review by emuvente — APPROVED** [review]
👍  looks good!

**Pull request merged**